ide/tasks.py

```diff
@@ -311,19 +311,22 @@ def isProcessPossible(layerId):
         json_str = json_str.strip("'<>() ").replace('\'', '\"')
         lrnLayer = imp.load_source('LRN', BASE_DIR + '/keras_app/custom_layers/lrn.py')
 
+        # clear clutter from previous graph built by keras to avoid duplicates
+        K.clear_session()
+
         model = model_from_json(json_str, {'LRN': lrnLayer.LRN})
 
-        sess = K.get_session()
-        tf.train.write_graph(sess.graph.as_graph_def(add_shapes=True), output_fld,
-                             output_file + '.pbtxt', as_text=True)
+        tf.train.export_meta_graph(
```

Comment:
@haabhi can you make the same changes in `tensorflow_app/views/export_grahdef.py`. As of now there are no test cases for celery tasks which is why we run tests on our older implementation and it's necessary to keep them synced.

Reply:
Alright, I am on it :)

Reply:
@Ram81 Done :+1: